### PR TITLE
Improve dropzone story

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium-oss/analysis-ui-components/issues/163

### CODE REVIEW

File: stories/dropZone.stories.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ export default {
   title: 'Layout/DropZone',
   args: {
     color: 'black',
+    // eslint-disable-next-line no-alert
+    onDrop: () => alert('files uploaded'),
```
Comment (lpatiny) [inline stories/dropZone.stories.tsx:11]: Is there a possibility to display the list of dropped files ?

Like in [this example](https://github.com/cheminfo/filelist-utils/blob/59e182c6649ebcf39df7880b47482190b18ac2b3/html/index.html).

Also could we display and edit the 'onDrop' callback in the storybook ?

**Review by lpatiny — CHANGES_REQUESTED** [review]

**Comment by lpatiny** [comment]
The goal here is really to quickly and easily be able to debug what is happening when you drop one file, a list of files or a folder. This is why we would need a callback that displays what was dropped, maybe in a JSON inspector if easy to do.

File: stories/dropZone.stories.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply stories/dropZone.stories.tsx:11]: > Is there a possibility to display the list of dropped files ?

yeah it's possible

**Comment by lpatiny** [comment]
Please also rebase this branch.

File: stories/dropZone.stories.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply stories/dropZone.stories.tsx]: > Is there a possibility to display the list of dropped files ?

table created.



> Also could we display and edit the 'onDrop' callback in the storybook ?

yes i added an onDrop action in the story

**Pull request merged**